### PR TITLE
Clarify that you must modify the general connection string with Sequelize.

### PR DESCRIPTION
**Pull request by ianjevans**
This PR clarifies the format of connection strings when using SQLAlchemy.

Closes DOC-6587.

### CODE REVIEW

File: css/bs/bootstrap-grid.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (nickvigilante) [inline css/bs/bootstrap-grid.css:1]: Can you give these files an extension of `.scss` instead and delete the error comments?

**Review by nickvigilante — CHANGES_REQUESTED** [review]
If you're in the office this week, let me know and we can work on this together.

**Comment by gemma-shay** [comment]
I can see in the convo history that you were working on this, but something weird is still going on with the code blocks. 
Screenshot 2023-02-02 at 3 22 56 PM

**Comment by ianjevans** [comment]
> I can see in the convo history that you were working on this, but something weird is still going on with the code blocks. Screenshot 2023-02-02 at 3 22 56 PM

Yes, the CSS change is now in #16174, which ideally we would merge before this PR so I can pull it in.

**Review by gemma-shay — APPROVED** [review]
gotcha, this lgtm!

**Pull request merged**